source/Playnite/Scripting/PowerShell/PowerShell.cs

```diff
@@ -50,6 +48,18 @@ public void Dispose()
             runspace.Dispose();
         }
 
+        public void ImportModule(string path)
+        {
+            powershell.Runspace.SessionStateProxy.Path.SetLocation(Path.GetDirectoryName(path));

```

Comment:
I guess setting the working directory of extensions is a breaking change too. I think they previously ran in the Playnite application directory. Maybe this can be removed, I haven't tested without it.

Reply:
This runs fine with these changes:
```powershell
$newGame = New-Object "Playnite.SDK.Models.Game"
$newGame.Name = "New Game"
$PlayniteApi.Database.Games.Add($newGame)
```